Docs: Split Traffic Analytics to a How-to guide and a Feature entry (Diátaxis) (#9677)
readthedocs/readthedocs.org: docs/user/reference/analytics.rst

Analytics for Search and Traffic
================================

Read the Docs supports analytics for search and traffic.
When someone reads your documentation,
we collect data about the vist and the referer
with full respect of the privacy of the visitor.

Traffic Analytics
-----------------

> Read the Docs aggregates simple statistics about visits to your documentation.

Reviewer: ```suggestion
Read the Docs aggregates statistics about visits to your documentation.
This is mainly information about how often pages are viewed,
and which return a `404 Not Found` error code. 
```

Traffic Analytics lets you see *which* documents your users are reading.
This allows you to understand how your documentation is being used,

> so you can focus on expanding and updating parts people are reading most.

Reviewer: 💯 benefit/value statement.

If you require more detailed analytics, Read the Docs has native support for Google Analytics.
It's also possible to customize your documentation to include other analytics frameworks.

Learn more in :doc:`/analytics`.

Search Analytics
----------------

When someone visits your documentation using a search engine,

> Read the Docs will collect analytics on their search term.

Reviewer: This is about in-doc search, not incoming search terms. Shows how little we've been doing a good job of documenting it.  😆

Author: :facepalm: 

I didn't continue writing for the exact reason of feeling like something was terribly wrong :D

Author: This has been clarified and a new how-to for search analytics has been created :+1:

> Analytics for Search and Traffic

Author: ```suggestion
Analytics for search and traffic
```

================================

Read the Docs supports analytics for search and traffic.
When someone reads your documentation,
we collect data about the vist and the referer
with full respect of the privacy of the visitor.

> Traffic Analytics

Author: ```suggestion
Traffic analytics
```
